content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1882 @@
+
+![The new blend_modes example.](blend-modes.png)
+
+**Note:** Meshes using the new blend modes are drawn on the existing `Transparent3d` render phase, and therefore the same _z-sorting considerations/limitations_ from `AlphaMode::Blend` apply.
+
+[`AlphaMode`]: https://docs.rs/bevy/0.10.0/bevy/pbr/enum.AlphaMode.html
+[`StandardMaterial`]: https://docs.rs/bevy/0.10.0/bevy/pbr/struct.StandardMaterial.html
+
+## More Tonemapping Choices
+
+<div class="release-feature-authors">authors: @DGriffin91, @JMS55</div>
+
+Tonemapping is the process of transforming raw High Dynamic Range (HDR) information into actual "screen colors" using a "display rendering transform" (DRT). In previous versions of Bevy you had exactly two tonemapping options: Reinhard Luminance or none at all. In **Bevy 0.10** we've added a ton of choices!
+
+### No Tonemapping
+
+This is generally not recommended as HDR lighting is not intended to be used as color.
+
+![no tonemapping](tm_none.png)
+
+### Reinhard
+
+A simple method that adapts to the color in a scene: `r = color / (1.0 + color)`. Lots of hue shifting, brights don't desaturate naturally. Bright primaries and secondaries don't desaturate at all.
+
+![reinhard](tm_reinhard.png)
+
+### Reinhard Luminance
+
+A popular method similar to normal Reinhard that incorporates luminance. It adapts to the amount of light in a scene. This is what we had in previous versions of Bevy. It is still our default algorithm, but this will likely change in the future. Hues shift. Brights don't desaturate much at all across the spectrum.
+
+![reinhard luminance](tm_reinhard_luminance.png)
+
+### ACES Fitted
+
+An extremely popular algorithm used in film and industry (ex: ACES is the default Unreal tonemapping algorithm). When people say "filmic", this is often what they mean.
+
+Not neutral, has a very specific aesthetic, intentional and dramatic hue shifting.
+Bright greens and reds turn orange. Bright blues turn magenta. Significantly increased contrast. Brights desaturate across the spectrum.
+
+![aces](tm_aces.png)
+
+### AgX
+
+Very neutral. Image is somewhat desaturated when compared to other transforms. Little to no hue shifting. Subtle [Abney shifting](https://en.wikipedia.org/wiki/Abney_effect). [Created by Troy Sobotka](https://github.com/sobotka/AgX)
+
+![agx](tm_agx.png)
+
+### SomewhatBoringDisplayTransform
```

Comment:
```suggestion
### Somewhat Boring Display Transform
```
a long unbreakable title means the ToC has a scrollbar

Reply:
Ooh good call!